compiler/src/dotty/tools/repl/Rendering.scala

```diff
@@ -40,6 +41,8 @@ private[repl] class Rendering(parentClassLoader: Option[ClassLoader] = None) {
       myClassLoader
     }
 
+  private[this] def MaxStringElements = 1000  // no need to mkString billions of elements
```

Comment:
Do you know what's used in scalac?

Reply:
Copy pasted: https://github.com/scala/scala/blob/ba9701059216c629410f4f23a2175d20ad62484b/src/repl/scala/tools/nsc/interpreter/MemberHandlers.scala#L127

Reply:
:shipit: